README.md

```diff
@@ -2,7 +2,7 @@
 
 ## Get an instance of SDL Core running
 
-Note: This requires you to use Ubuntu 16.04 or 18.04.
+Note: This requires you to use Ubuntu 16.04, 18.04, or 20.04.
```

Comment:
```suggestion
Note: This requires you to use Ubuntu 16.04, 18.04 or 20.04.
```

Reply:
This is used in pretty much every other part of our documentation, I see no benefit in removing it here